### PR TITLE
Added comments endpoint

### PR DESCRIPTION
**Pull request by zaneris123**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -120,4 +120,49 @@ describe("GET, Reviews",()=>{
...
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    review_id: 2
+                    })
+                })
+            })
+    })
+    test("200: Recieve empty array",()=>{
```
Comment (callumu21) [inline __tests__/api.test.js:144]: Just be careful to keep these descriptive enough - 'Receive an empty array for a review with no comments'. Great job including it, though, as this is the one that's often forgotten!

File: __tests__/api.test.js [hunk 2]
```diff
@@ -120,4 +120,49 @@ describe("GET, Reviews",()=>{
...
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    review_id: 2
+                    })
+                })
+            })
+    })
```
Comment (callumu21) [inline __tests__/api.test.js:143]: Just need to add a test or assertion that checks the comments are returned in reverse chronological order (i.e. most recent first)

File: controllers/comments.controller.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const {fetchCommentsByReview} = require("../models/comments.model.js")
+const {fetchReview} = require("../models/reviews.model.js")
+
+exports.getCommentsByReview = (req, res, next) => {
+    fetchReview(req.params.id)
```
Comment (callumu21) [inline controllers/comments.controller.js:5]: Just be careful with running multiple async functions, as we need to ensure that the response is only sent once both are successfully run. Can you chain your .thens to ensure that the response is only sent once everything has been run? Alternatively, you may want to look at Promise.all, if that's a more comfortable syntax for you

**Review by callumu21 — CHANGES_REQUESTED** [review]
Really great start! Just missing a test to check that the comments are returned with the most recent first. I'd also look at refactoring your controller to ensure that your two async functions have finished being executed before sending a response

**Review by HeatherEver — APPROVED** [review]
Nice refactor, go ahead and merge

**Pull request merged**